src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,72 @@
 'use strict';
 
+const NAME = '';
+const HEALTH = 100;
+
 class Animal {
-  // write your code here
+  static alive = [];
+
+  health;
+  name;
+
+  constructor(name = NAME, health = HEALTH) {
+    this.name = name;
+    this.health = health;
+
+    if (this.health > 0) {
+      Animal.alive.push(this);
+    }
```

Comment:
There's no need to check if the health is greater than 0 upon creation of an animal since the default health is set to 100. All animals should be alive when created.